seatunnel-dist/src/test/java/org/apache/seatunnel/api/file/AllFileSpecificationCheckTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.api.file;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileVisitOption;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+@Slf4j
+@DisabledOnOs(OS.WINDOWS)
+public class AllFileSpecificationCheckTest {
+
+    private static Map<String, List<String>> fileContents;
+
+    @BeforeAll
+    public static void beforeAll() throws IOException {
+        List<String> fileTypesCanNotRead =
+                Arrays.asList("parquet", "orc", "xlsx", "xls", "png", "jar", "lzo", "zip", "ico");
+        List<String> fileCanNotRead =
+                Arrays.asList(
+                        "seatunnel-connectors-v2/connector-file/connector-file-base/src/test/resources/encoding/gbk.json",
+                        "seatunnel-connectors-v2/connector-file/connector-file-base/src/test/resources/encoding/gbk.xml",
+                        "seatunnel-connectors-v2/connector-file/connector-file-base/src/test/resources/encoding/gbk_use_attr_format.xml",
+                        "seatunnel-connectors-v2/connector-file/connector-file-base/src/test/resources/encoding/gbk.txt",
+                        "seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/json/e2e_gbk.json",
+                        "seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/text/e2e_gbk.txt");
```

Comment:
why？

Reply:
Can not read by UTF-8 encode. 
![image](https://github.com/user-attachments/assets/ba3cb3e8-1d91-4371-9f2e-941554bc01e6)